File: Example/CoreData/SyncKitCoreDataExampleTests/QSTestEntity+CoreDataProperties.m
//
//  QSTestEntity+CoreDataProperties.m
//  SyncKitCoreDataExampleTests
//
//  Created by Manuel Entrena on 18/10/2018.
//  Copyright © 2018 Manuel. All rights reserved.
//
//

#import "QSTestEntity+CoreDataProperties.h"

@implementation QSTestEntity (CoreDataProperties)

+ (NSFetchRequest<QSTestEntity *> *)fetchRequest {
	return [NSFetchRequest fetchRequestWithEntityName:@"QSTestEntity"];
}

@dynamic identifier;
@dynamic names;

@end

File: Example/CoreData/SyncKitCoreDataExample/SyncKitCoreDataExampleTests/Transformable/QSTestEntity+CoreDataProperties.swift
//
//  QSTestEntity+CoreDataProperties.swift
//  SyncKitCoreDataExampleTests
//
//  Created by Manuel Entrena on 18/06/2019.
//  Copyright © 2019 Manuel Entrena. All rights reserved.
//
//

import Foundation
import CoreData


extension QSTestEntity {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<QSTestEntity> {
        return NSFetchRequest<QSTestEntity>(entityName: "QSTestEntity")
    }

    @NSManaged public var identifier: String?
    @NSManaged public var names: NSArray?

}
